otel-infrastructure-collector/README.md

```diff
@@ -0,0 +1,84 @@
+
+The OpenTelemetry collector offers a vendor-agnostic implementation of how to receive, process and export telemetry data. 
+In this chart, the collector will be deployed as a single replica deployment. 
+
+The Infrastructure collector provides:
+
+- [Coralogix Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/coralogixexporter) -  Coralogix exporter is preconfigured to enrich data using Kubernetes Attributes, which allows quick correlation of telemetry signals using consistent ApplicationName and SubsytemName fields.
+- [Cluster Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver) - The Kubernetes Cluster receiver collects cluster-level metrics from the Kubernetes API server. Alternative to Kube State Metrics project.
+
+## Prerequisites
+
+### Secret Key
+
+Follow the [private key docs](https://coralogix.com/docs/private-key/) tutorial to obtain your secret key tutorial to obtain your secret key.
+
+OpenTelemetry Agent require a `secret` called `coralogix-keys` with the relevant `private key` under a secret key called `PRIVATE_KEY`, inside the `same namespace` that the chart is installed in.
+
+
+```bash
+kubectl create secret generic coralogix-keys \
+  --from-literal=PRIVATE_KEY=<private-key>
+```
+
+The created secret should look like this:
+```yaml
+apiVersion: v1
+data:
+  PRIVATE_KEY: <encrypted-private-key>
+kind: Secret
+metadata:
+  name: coralogix-keys
+  namespace: <the-release-namespace>
+type: Opaque 
+```
+
+### Endpoints
+
+### Coralogix's Endpoints 
+
+Depending on your region, you need to configure correct Coralogix endpoint. Here are the available Endpoints:
+
+| Region  | Traces Endpoint                          | Metrics Endpoint                     | Logs Endpoint                     |
+|---------|------------------------------------------|------------------------------------- | --------------------------------- |
+| USA1    | `otel-traces.coralogix.us:443`      | `otel-metrics.coralogix.us:443`      | `otel-logs.coralogix.us:443`      |
+| APAC1   | `otel-traces.app.coralogix.in:443`  | `otel-metrics.coralogix.in:443`      | `otel-logs.coralogix.in:443`      | 
+| APAC2   | `otel-traces.coralogixsg.com:443`   | `otel-metrics.coralogixsg.com:443`   | `otel-logs.coralogixsg.com:443`   |
+| EUROPE1 | `otel-traces.coralogix.com:443`     | `otel-metrics.coralogix.com:443`     | `otel-logs.coralogix.com:443`     |
+| EUROPE2 | `otel-traces.eu2.coralogix.com:443` | `otel-metrics.eu2.coralogix.com:443` | `otel-logs.eu2.coralogix.com:443` |
+
+Example configuration:
+```yaml
+#values.yaml:
+---
+global:
+  traces:
+    endpoint: "<traces-endpoint-here>"
+  metrics:
+    endpoint: "<metrics-endpoint-here>"
+  logs:
+    endpoint: "<logs-endpoint-here>"
+```  
+
+## Installation
+
+```bash
+helm upgrade --install otel-coralogix-agent coralogix-charts-virtual/opentelemetry-coralogix \
```

Comment:
otel-infrastructure-collector instead of opentelemetry-coralogix, 
and maybe we should put a different name to differ it from the agent mode collector